bigbluebutton-html5/imports/api/audio/client/bridge/sip.js

```diff
@@ -324,7 +325,21 @@ class SIPSession {
       if (browser().name === 'edge') {
         connectionCompletedEvents = ['iceConnectionCompleted'];
       }
-
+      //getStats() API
+      const getWebRTCStats = (peer) => { 
+        (async () => {
+          const report = await peer.getStats();
+          let allRtcStats = new Array();
+          for (let item of report.values())                          
+            allRtcStats.push(item);
+          let result = {};                           
+          result['enabled']= this.webrtcConnected;
+          result['timer']=  Date.now();
+          result['rtcStats']= allRtcStats;
```

Comment:
The variable should a constant because it doesn't change and it should be:
```
const result = {
  enabled: this.webrtcConnected,
  rtcStats: allRtcStats,
};
```